command/create_game.rb

```diff
@@ -43,6 +43,10 @@ def execute(context:)
 
       Stats.create!(player_id: player.id, highest_hp: player.hp, highest_range: player.range)
 
+      if GlobalStats.find_by(player_discord_id: user.id).nil?
+        GlobalStats.create!(player_discord_id: user.id, username: user.username, highest_hp: player.hp, highest_range: player.range)
+      end
```

Comment:
Does it create most_cities_hsld when it does the check in capture_city?

Reply:
Same with energy given

Reply:
Yes and yes :)

Reply:
✌️